todo_app/data/trello_items.py

```diff
@@ -0,0 +1,88 @@
+import os, requests
+from flask import request
+
+trello_url = os.getenv("BASE_URL")
+trello_key = os.getenv("API_KEY")
+trello_token = os.getenv("TOKEN")
+
+trello_board_id = os.getenv("BOARD_ID")
```

Comment:
As mentioned on the call yesterday - this is great and works but may cause you headaches when it comes to the next module due to the order in which your code gets run. One simple approach here might be:

1) Convert each of these variable loads to a function, like:
```python
def trello_url():
    return os.getenv("BASE_URL")
```
And then swap anywhere that uses them from `trello_url` to `trello_url()`

2) Change the list loading to an explicit "init" function:
```python
todo_idlist = None
doing_idlist = None
done_idlist = None

def load_lists():
    get_lists_path = "boards/"+trello_board_id() +"/lists"
    url = trello_url + get_lists_path
    query_params = {"key": trello_key(), "token": trello_token(), "field": "name"}
    print(f"--> URL: {url}")
    response = requests.request("GET", url, headers=headers, 
    params=query_params)
    response_json = response.json()
    print(response_json)


    # this line allows us to change the variables defined outside the function
    global todo_idlist, doing_idlist, done_idlist

    todo_idlist = response_json[0].get("id")
    doing_idlist = response_json[1].get("id")
    done_idlist = response_json[2].get("id")
```
And then execute load_lists early in the app.py file - we can then move that inside the `create_app` function when we add one during M3

Having said all the above - it likely makes sense to make these changes as part of M3 rather than immediately here

Reply:
A totally different alternative to part 2 is to simply ask users to provide three extra environment variables with the values of those lists - whether this is "better" or not is definitely up for discussion, but it can make the code quite a bit simpler